playbooks/kubevirt.yml

```diff
@@ -0,0 +1,253 @@
+        fstype: swap
+        path: none
+        state: absent
+    - name: modprobe br_netfilter
+      modprobe:
+        name: br_netfilter
+
+    - name: configure bridge iptables
+      sysctl:
+        name: "{{ item }}"
+        value: 1
+        sysctl_file: /etc/sysctl.d/k8s.conf
+      with_items:
+        - net.bridge.bridge-nf-call-ip6tables
+        - net.bridge.bridge-nf-call-iptables
+
+    - name: install docker
+      yum:
+        name: docker
+
+    - name: enable docker
+      service:
+        name: docker
+        enabled: true
+        state: started
+
+    - name: enable CentOS OpenStack repository
+      yum_repository:
+        name: centos-openstack-rocky
+        description: CentOS OpenStack
+        baseurl: http://mirror.centos.org/centos/$releasever/cloud/$basearch/openstack-rocky/
+        enabled: true
+        gpgcheck: false
+
+    - name: install openvswitch
+      yum:
+        name: openvswitch
+
+    - name: enable openvswitch
+      service:
+        name: openvswitch
+        enabled: true
+        state: started
+
+    - name: add foreman bridge
+      openvswitch_bridge:
+        bridge: foreman
+        state: present
+
+    - name: add port to foreman bridge
+      openvswitch_port:
+        bridge: foreman
+        port: eth1
+        state: present
+
+
+    - name: k8s repo
+      yum_repository:
+        name: kubernetes
+        description: Kubernetes
+        baseurl: https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
+        enabled: true
+        gpgcheck: true
+        repo_gpgcheck: true
+        gpgkey: https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+        exclude: kube*
+
+    - name: install kubelet kubeadm kubectl
+      yum:
+        name:
+          - kubelet
+          - kubeadm
+          - kubectl
+        disable_excludes: kubernetes
+
+    - name: enable kubelet accounting
+      copy:
+        dest: /etc/systemd/system/kubelet.service.d/11-cgroups.conf
+        content: |
+          [Service]
+          CPUAccounting=true
+          MemoryAccounting=true
+
+    - name: reload systemd
+      systemd:
+        daemon_reload: yes
+
+    - name: enable kubelet
+      service:
+        name: kubelet
+        enabled: true
+        state: started
+
+    - name: init cluster
+      command: kubeadm init --pod-network-cidr={{ pod_network }} --apiserver-advertise-address={{ ansible_eth0['ipv4']['address'] }}
+      args:
+        creates: /etc/kubernetes/admin.conf
+
+    - name: untaint master
+      command: kubectl taint nodes --all node-role.kubernetes.io/master-
+      register: untaint_master
+      failed_when: false
+      changed_when: untaint_master.rc == 0
+      environment:
+        KUBECONFIG: /etc/kubernetes/admin.conf
+
+    - name: deploy multus
+      command: kubectl apply -f https://raw.githubusercontent.com/intel/multus-cni/{{ multus_version }}/images/multus-daemonset.yml
```

Comment:
This manifest do not deploy multus correctly. With this configuration VMI fail to run with:

> "Failed to sync vmi","name":"vmi-multus","namespace":"default","pos":"server.go:123","reason":"preparing the pod network failed: Link not found"

Reply:
I will look for a way from where we can obtain correct manifest. Maybe we can store it together with the plugin.